tests/plugins/test_build_source_container.py

```diff
@@ -80,10 +85,75 @@ def test_running_build(tmpdir, caplog):
             'args': {},
         }]
     )
-    build_result = runner.run()
-    assert not build_result.is_failed()
-    assert build_result.oci_image_path
-    assert 'stub stdout' in caplog.text
+
+    temp_source_dir = os.path.join(str(tmpdir), 'source_dir')
+    temp_image_output_dir = os.path.join(str(tmpdir), 'image_output_dir')
+    temp_image_export_dir = os.path.join(str(tmpdir), 'image_export_dir')
+    tempfile_chain = flexmock(tempfile).should_receive("mkdtemp").and_return(temp_source_dir)
+    tempfile_chain.and_return(temp_image_output_dir)
+    tempfile_chain.and_return(temp_image_export_dir)
+    os.mkdir(temp_source_dir)
+    os.mkdir(temp_image_export_dir)
+    os.makedirs(os.path.join(temp_image_output_dir, 'blobs', 'sha256'))
+
+    def check_check_output(args, **kwargs):
+        if args[0] == 'skopeo':
+            assert args[0] == 'skopeo'
+            assert args[1] == 'copy'
+            assert args[2] == 'oci:%s' % temp_image_output_dir
+            assert args[3] == 'oci-archive:%s' % os.path.join(temp_image_export_dir,
+                                                              EXPORTED_SQUASHED_IMAGE_NAME)
+
+            if export_failed:
+                raise subprocess.CalledProcessError(returncode=1, cmd=args, output="Failed")
+
+            return ''
+        else:
+            assert args[0] == 'bsi'
+            assert args[1] == '-d'
+            assert args[2] == 'sourcedriver_rpm_dir'
+            assert args[3] == '-s'
+            assert args[4] == temp_source_dir
+            assert args[5] == '-o'
+            assert args[6] == temp_image_output_dir
+            return 'stub stdout'
+
+    (flexmock(subprocess)
+     .should_receive("check_output")
+     .times(2)
+     .replace_with(check_check_output))
+
+    blob_sha = "f568c411849e21aa3917973f1c5b120f6b52fe69b1944dfb977bc11bed6fbb6d"
+    index_json = {"schemaVersion": 2,
+                  "manifests":
+                      [{"mediaType": "application/vnd.oci.image.manifest.v1+json",
+                        "digest":
+                        "sha256:%s" % blob_sha,
```

Comment:
Can we put the value on the same line as the key?

Reply:
sure